Equinox/Globals.h

```diff
@@ -54,5 +56,68 @@ typedef unsigned int uint;
 #define RESIZABLE true
 #define TITLE "Equinox Engine Editor"
 
+inline void DrawBoundingBox(AABB& boundingBox)
```

Comment:
Globals is the place where we can define utilities that are common (or will be useful) for the code.
It doesn't make sense to create a module, create its update, start, clean, etc, methods, and call the APP in the code when you just need a function that does/returns simple logic.

That's not the way an utility works.